Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -102,8 +116,17 @@ private void DispatchToValidationFailureRepository (IBusinessObjectValidationRes
 
     protected override bool EvaluateIsValid ()
     {
-      // TODO RM-6056: Shows a validation error if IBusinessObjectValidationResult returned messages for this control.
-      return true;
+      var sb = new StringBuilder();
+
+      foreach (var validationFailure in _validationFailuresForBusinessObjectPropertyOfBocList)
```

Comment:
We should source the validation failures from the repo using `GetUnhandledValidationFailuresForBocList` instead. These are the failures that we want to display anyway so this is fine. If there are any other failures we should also add another line with a "BocList has validation failures" text. If you want we can also use the static text for now and move the formatting to the other issue.